apps/opik-frontend/src/components/pages/FeedbackDefinitionsPage/FeedbackDefinitionsActionsPanel.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useRef, useState } from "react";
+import { Trash } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+import { FeedbackDefinition } from "@/types/feedback-definitions";
+import useFeedbackDefinitionBatchDeleteMutation from "@/api/feedback-definitions/useFeedbackDefinitionBatchDeleteMutation";
+import ConfirmDialog from "@/components/shared/ConfirmDialog/ConfirmDialog";
+import TooltipWrapper from "@/components/shared/TooltipWrapper/TooltipWrapper";
+
+type FeedbackDefinitionsActionsPanelsProps = {
+  feedbackDefinitions: FeedbackDefinition[];
+};
+
+const FeedbackDefinitionsActionsPanel: React.FunctionComponent<
+  FeedbackDefinitionsActionsPanelsProps
+> = ({ feedbackDefinitions }) => {
+  const resetKeyRef = useRef(0);
+  const [open, setOpen] = useState<boolean>(false);
+  const disabled = !feedbackDefinitions?.length;
+
+  const { mutate } = useFeedbackDefinitionBatchDeleteMutation();
+
+  const deleteFeedbackDefinitionsHandler = useCallback(() => {
+    mutate({
+      ids: feedbackDefinitions.map((f) => f.id),
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [feedbackDefinitions]);
```

Comment:
i would add `mutate` to the dependencies

Reply:
Fixed